dbt/redshift/redshift_etl.py

```diff
@@ -0,0 +1,127 @@
+from __future__ import absolute_import
+import logging
+import os
+import psycopg2
+import subprocess
+from dateutil import rrule
+
+from parsely_raw_data import redshift as parsely_redshift
+from parsely_raw_data import utils as parsely_utils
+from dbt.redshift.settings.default import *
+from dbt.redshift.settings.merge_settings_yaml import migrate_settings
+
+SETTINGS_ARG_MAPPING = {
+    'table_name': PARSELY_RAW_DATA_TABLE,
+    'host': REDSHIFT_HOST,
+    'user': REDSHIFT_USER,
+    'password': REDSHIFT_PASSWORD,
+    'database': REDSHIFT_DATABASE,
+    'port': REDSHIFT_PORT,
+    'keep_extra_data': ETL_KEEP_RAW_DATA,
+    'network': S3_NETWORK_NAME,
+    'access_key_id': S3_AWS_ACCESS_KEY_ID,
+    'secret_access_key': S3_AWS_SECRET_ACCESS_KEY,
+    'start_date': ETL_START_DATE,
+    'end_date': ETL_END_DATE,
+    'dbt_profiles_dir': DBT_PROFILE_LOCATION,
+    'dbt_target': DBT_PROFILE_TARGET_NAME
+}
+
+
+def get_settings_arg_mapping_value(field_name, arg_value):
+    if arg_value:
+        return arg_value
+    else:
+        settings_value = SETTINGS_ARG_MAPPING[field_name]
+        return settings_value
```

Comment:
you could simplify this to:

```
def get_settings_arg_mapping_value(field_name, arg_value):
    return arg_value or SETTINGS_ARG_MAPPING[field_name]
```

since x or y in python will return x if x is truthy otherwise y